frontend/src/scenes/exports/ExportsList.tsx

```diff
@@ -103,6 +104,16 @@ export function Exports(): JSX.Element {
                                 error: resumeError,
                             } = useExportAction(currentTeamId, export_.id, 'unpause')
 
+                            const deleteCallback = useCallback(() => {
+                                exports.pop(index)
```

Comment:
There's likely something we should do here around ensuring that the relevant components are notified of the state change here in a more deliberate way.

Perhaps inefficient but we could simply:

1. return an additional callback, say `refresh` or `refetch` from `useExports`
2. call this within the `deleteExport().then(...)`

Another option could be to do the state update optimistically, but more deliberately by:

1. returning an additional callback, say `update` from `useExports`, which would explicitly call the set state and thus trigger a rerender.
2. call this within the `deleteExport().then(...)`

If it works for now I think it's fine. Would be great to have a test for it but also that can go in a separate followup, it's not like we have people depending on this atm.

Reply:
I went with option number 2. And also took your suggestion of adding a Toast (see the new demo gif).

Reply:
I'll follow up with more toasts, changes to the individual export UI, and tests.